db/migrate/20161215161941_add_enabled_field_to_ems.rb

```diff
@@ -0,0 +1,7 @@
+class AddEnabledFieldToEms < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base; end
+
+  def change
+    add_column :ext_management_systems, :enabled, :boolean, :null => false, :default => true
```

Comment:
Please remove the `null` and `default` keys and prefer `default_value_for` in the model.  Instead prefer a data migration.